src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int MAX_NUMBER = 5;

```

Comment:
Do not use the constant for this. Use Color.values().length
If the Color enum is changed, this MAX_NUMBER will become incorrect.